.github/scripts/test-all.sh

```diff
@@ -17,7 +17,7 @@ echo "$packages" | tr ' ' '\n' | while read -r item; do
   echo "running $item"
 
   case "$item" in
-    *".github"*|*"experimental"*|*"deployment-platforms"*)
+    *".github"*|*"experimental"*|*"deployment-platforms"*|*"databases"*)
```

Comment:
Why do we need to disable tests for this?

Reply:
In this PR  the `schema.prisma` is empty which causes the [test to fail](https://github.com/prisma/prisma-examples/runs/1314708843#step:5:68). 

I'm probably going to add a test for both introspection and generation.

Reply:
There's a separate workflow I will push which tests this example exclusively, so I'm leaving this here

Reply:
This is a lot of added complexity. Please open a new issue for @divyenduz to look at in the future if we can unify that again with our main test setup in some way. Otherwise we can not guarantee the same confidence in this working with `dev` etc.